cli/cli-core/src/main/java/io/telicent/smart/cache/cli/options/KafkaOptions.java

```diff
@@ -210,11 +215,16 @@ public <TKey, TValue> KafkaReadPolicy<TKey, TValue> toReadPolicy() {
                 case EARLIEST -> KafkaReadPolicies.fromEarliest();
                 case LATEST -> KafkaReadPolicies.fromLatest();
                 case END -> KafkaReadPolicies.fromEnd();
+                case EXTERNAL -> getExternalReadPolicy();
                 default -> KafkaReadPolicies.fromBeginning();
             };
         }
     }
 
+    public static <TKey, TValue> KafkaReadPolicy<TKey, TValue> getExternalReadPolicy() {
+        throw new IllegalArgumentException("External Read Policy is not supported");
```

Comment:
Note that there is already a `KafkaReadPolicies.fromExternalOffsets()` that provides an appropriate read policy, however that does need passing in the `OffsetStore` and a `defaultOffset` so we'd need to change this method and the type signature on `toReadPolicy()` to accept extra arguments if we want `EXTERNAL` to be a supported option